runtime/queries/html/textobjects.scm

```diff
@@ -0,0 +1,5 @@
+(element) @xml_element.around
+
+((start_tag) 
+	[(text) (element) ]* @xml_element.inside
+(end_tag))
```

Comment:
let's use something like

```scm
(element
  (start_tag)
  (_)* @xml_element.inside
  (end_tag)) @xml_element.around

(start_tag) @xml_element.around

(comment) @comment.around
```

for html. That handles self-closing elements like `<link ... />`

Plus might as well throw the comment textobject in there while we're making textobjects 😛

Reply:
added the queries for self_closing_tags,
also sorry but i don't understand why do you have that anchor at the end of the first query
is it some case that i am missing to see?

Reply:
Ah actually I forgot about `self_closing_tag`. The anchor there is so that the stanza matches only elements with both start and end tag. The second stanza then matches elements with just a `start_tag` like how peolpe sometimes write `<br>`

Reply:
I suppose that the grammar rules ensure that an element can only have one trailing `end_tag` though, right? So I suppose there's no need for the anchor